### PR TITLE
Initial open source release of MiniGo implementation.

### PR DESCRIPTION
**Pull request by saeta**

### CODE REVIEW

**Comment by saeta** [comment]
@xiejw ^^^

File: MiniGo/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# MiniGo
+
+This folder is the Swift ([S4TF]) verison of the [MiniGo] game.
+
+## To Get a MiniGo Checkpoint
+
+Currently the inference uses a pre-trained checkpoint. To get it, run
+
+    mkdir -p MiniGo
+    gsutil cp gs://minigo-pub/v15-19x19/models/000939-heron.* MiniGo/
```
Comment (xiejw) [inline MiniGo/README.md]: make sure this gcd is accessible

File: MiniGo/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+To compile and run the code, make sure you have the latest S4TF toolchain (see
+next section).
+
+Then, follow the following steps:
+
+    swift test
+    swift run
+
+## Swift for TensorFlow Toolchain
```
Comment (xiejw) [inline MiniGo/README.md]: Let's delete all the line after here.

File: MiniGo/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+In order to use [Swift Package Manager][SwiftPM], a toolchain is required. You
+can
+
+1. download the pre-built package from [official
+   site](https://github.com/tensorflow/swift/blob/master/Installation.md),
+2. compile the toolchain by your own, or
```
Comment (xiejw) [inline MiniGo/README.md]: s/, or/. 

end the sentence there.

File: MiniGo/README.md (hunk 1 above)
Comment (saeta) [reply MiniGo/README.md]: Confirmed with my personal account. Thanks!

File: MiniGo/README.md (hunk 3 above)
Comment (saeta) [reply MiniGo/README.md]: Yeah, sorry, just noticed that. Should be fixed up now.

File: MiniGo/README.md (hunk 2 above)
Comment (saeta) [reply MiniGo/README.md]: Done!

**Pull request merged**